anesthetic/samples.py

```diff
@@ -206,33 +206,19 @@ def plot_2d(self, axes, *args, **kwargs):
         types: dict, optional
             What type (or types) of plots to produce. Takes the keys 'diagonal'
             for the 1D plots and 'lower' and 'upper' for the 2D plots.
-            The options for 'diagonal are either:
+            The options for 'diagonal are:
```

Comment:
If you dislike the 'either', should it be removed for the parameter `axes` as well?

Reply:
Indeed. I've also cleaned up some of the other lists in the documentation of this file